### ISSUE TITLE
Fields of struct are treated as globals

### ISSUE THREAD

**Issue by tkf** [JuliaStaging/JuliaVariables.jl#9]
Hi, thanks for a super useful package. Unfortunately, I think I find a bug

```julia
julia> solve(:(struct S
           x
       end))
:(struct @global S
      #= REPL[11]:2 =#
      @global x
  end)
```

I suppose `@global x` is wrong?

**Comment by thautwarm** [comment]
Yes, I didn't treat struct yet.
A bit curious, Is this necessary? I think the scope of struct is simple so we can treat them separately.

**Comment by tkf** [comment]
Yeah, I think I can skip fields while processing `solve`d tree.  But I thought it would be nice if I can just blindly look for `ScopedVar`.
